code/modules/mining/coins.dm

```diff
@@ -44,6 +44,28 @@
 	name = "platinum coin"
 	icon_state = "coin_adamantine"
 
+/obj/item/weapon/coin/erp
+	name = "erp token"
+	icon_state = "erp_token"
+	desc = "Put this in her slot to redeem your erp token. Good for 1 deciseconds of erp."
```

Comment:
Description is somewhat deceptive, as it implies ERP token's can only be used on females. If this is true (which is a little biased if you ask me), please ensure the code reflects this. Possibly a dynamic description based on SQL database statistics determining the most likely recipient for an ERP token tracked over indefinite intervals of time?